src/seedu/addressbook/commands/SortCommand.java

```diff
@@ -0,0 +1,4 @@
+package seedu.addressbook.commands;
+
+public class SortCommand {
```

Comment:
Perhaps you could refine the commit subject title and message body further to indicate that this is an unused file!